assembl/static2/js/app/components/common/headerStatistics.jsx

```diff
@@ -40,22 +40,24 @@ export const statParticipations = (numParticipations: number): StatisticElementP
   metricNameTranslateKey: 'home.participations'
 });
 
-const StatisticElement = (props: StatisticElementProps) => (
+const StatisticElement = ({ metricNameTranslateKey, iconName, metricValue }: StatisticElementProps) => (
   <div className="stat-container">
     <div className="stat-box">
-      <div className={`stat-icon assembl-icon-${props.iconName} white`} />
+      <div className={`stat-icon assembl-icon-${iconName} white`} />
       <div className="stat">
-        <div className="stat-nb">{props.metricValue}</div>
+        <div className="stat-nb">
+          <Localize value={metricValue} />
+        </div>
         <div className="stat-nb stat-label">
-          <Translate value={props.metricNameTranslateKey} count={props.metricValue} />
+          <Translate value={metricNameTranslateKey} count={metricValue} />
         </div>
       </div>
     </div>
   </div>
 );
 
 const mapElementsPropsToComponents = elemsProps =>
-  elemsProps.map((elementProps, index) => <StatisticElement key={index} {...elementProps} />);
+  elemsProps.map((elementProps, index) => <StatisticElement key={`stat-${index}`} {...elementProps} />);
```

Comment:
you can use elemsProps.metricNameTranslateKey as key instead to avoid the eslint warning